src/Polymesh.ts

```diff
@@ -136,13 +136,20 @@ export class Polymesh {
       middleware?: MiddlewareConfig;
     }
   ): Promise<Polymesh> {
-    const { nodeUrl, accountSeed, keyring, accountUri, accountMnemonic, signer, middleware } =
-      params;
+    const {
+      nodeUrl,
+      accountSeed,
+      keyring,
+      accountUri,
+      accountMnemonic,
+      signer,
+      middleware,
+    } = params;
     let context: Context;
 
     /* istanbul ignore next: part of configuration, doesn't need to be tested */
     const wsp = new WebSocketAsPromised(nodeUrl, {
-      createWebSocket: url => new W3CWebSocket(url) as unknown as WebSocket,
+      createWebSocket: url => (new W3CWebSocket(url) as unknown) as WebSocket,
```

Comment:
FYI: I bumped the prettier version, so merging alpha, running `yarn` and then `yarn format` should clean up the diff